TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java

```diff
@@ -0,0 +1,37 @@
+package tests.data_profiles;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.DashboardPage;
+import page_objects.DataProfilesPage;
+import page_objects.LoginPage;
+import tests.BaseTest;
+import utils.common.Constants;
+import utils.helper.Logger;
+
+public class PositiveDPTests extends BaseTest {
+    LoginPage loginPage = new LoginPage();
+    DataProfilesPage dataProfilesPage = new DataProfilesPage();
+    DashboardPage dashboardPage = new DashboardPage();
+
+    @Test(description = "Verify that all Pre-set Data Profiles are populated correctly")
+    public void tc066_PreSetDataProfilesArePopulatedCorrectly() {
+        Logger.step("Login with a valid user");
+        loginPage.login("administrator", "");
```

Comment:
why don't you have constant for `administrator`?

Reply:
OK